src/somoclu.cpp

```diff
@@ -54,6 +54,7 @@ void printUsage() {
          "Arguments:\n" \
          "     -c FILENAME           Specify an initial map.codebook for the map.\n" \
          "     -d NUMBER             Coefficient in the Gaussian neighborhood function exp(-||x-y||^2/(2*(coeff*radius)^2)) (default: 0.5)\n" \
+         "     -D NORM               Norm to be used among vectors: euclidean, norm-inf or norm-p with any real p > 0\n" \
```

Comment:
add a (default) here?

Reply:
the default is already there, clarified in the 3rd commit I've just added in the patch-set.